src/ui/public/tooltip/tooltip.js

```diff
@@ -17,14 +17,18 @@
  * under the License.
  */
 
+import 'ui/angular-bootstrap';
 import html from './tooltip.html';
 import chrome from 'ui/chrome';
+import  { uiModules } from 'ui/modules';
 
-require('ui-bootstrap')
+uiModules.get('kibana', ['ui.bootstrap'])
```

Comment:
I think we can move this `['ui.bootstrap']` dependency to the global list we're moving to `chrome.setupAngular()`.

Reply:
Solved accordingly what we agreed in the zoom call 😃